### PR TITLE
update doc for iap

### LINKED ISSUE #852: [GCP] Update ingress troubleshooting/monitoring page for 0.6
The page
https://www.kubeflow.org/docs/gke/deploy/monitor-iap-setup/

Needs to be updated for 0.6

* We are now using ISTIO
* Ingress is in the istio-system namespace

### PR DESCRIPTION
**Pull request by lluunn**
fix #852 . 

/cc @jlewi @kunmingg

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/883)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-883--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/monitor-iap-setup/

File: content/docs/gke/deploy/monitor-iap-setup.md [hunk 1]
```diff
@@ -153,61 +153,34 @@ problems:
...
-    * A small number of restarts is expected since the configuration process
-      restarts the Envoy containers.
-
-    * If the pods are crash looping look at the logs to try to figure out why.
-
-        ```
-        kubectl -n kubeflow logs ${POD}
-        ```
+    The service with port `31380` is the one that handles Kubeflow 
+    traffic. (31380 is the default port of the service istio-ingressgateway)
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/monitor-iap-setup.md]: Add code formatting and punctuation:

```
(31380 is the default port of the service `istio-ingressgateway`.)
```

File: content/docs/gke/deploy/monitor-iap-setup.md [hunk 2]
```diff
@@ -153,61 +153,34 @@ problems:
...
 
-        Refer to the [troubleshooting 
-        guide](/docs/gke/troubleshooting-gke/#envoy-pods-crash-looping-root-cause-is-backend-quota-exceeded)
-        for common problems, including exceeded quota.
+    If the backend is unhealthy, check the pods in istio-system
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/monitor-iap-setup.md]: Add code formatting and punctuation:

```
If the backend is unhealthy, check the pods in `istio-system`:
```

File: content/docs/gke/deploy/monitor-iap-setup.md [hunk 3]
```diff
@@ -153,61 +153,34 @@ problems:
...
+    * kubectl get pods -n istio-system
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/monitor-iap-setup.md]: Add code formatting:

```
    * `kubectl get pods -n istio-system`
```

File: content/docs/gke/deploy/monitor-iap-setup.md [hunk 4]
```diff
@@ -153,61 +153,34 @@ problems:
...
+    * The `istio-ingressgateway-XX` ones should be running
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/monitor-iap-setup.md]: s/ones/pods

**Comment by lluunn** [comment]
Thanks @sarahmaddox , PTAL

**Comment by sarahmaddox** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**